modules/@apostrophecms/schema/ui/apos/components/AposSchema.vue

```diff
@@ -139,4 +141,9 @@ export default {
       }
     }
   }
+
+  :deep(.apos-field__wrapper--highlight > .apos-field) {
+    padding: 10px;
+    background: #FFF6E8;
```

Comment:
IMHO we should create a new variable to store the color in `_theme.scss`

Reply:
Done